docs/language/global-sphinx-files/_templates/layout.html

```diff
@@ -11,8 +11,9 @@
 {%- extends "alabaster/layout.html" %}
 
 {%- block extrahead %}
-  <script async="" src="https://www.google-analytics.com/analytics.js"></script>
-  <script src="https://help.semmle.com/GAtracking.js"></script>
+  <!-- Google Tag Manager -->
+  <script src="https://help.semmle.com/GAtracking.js"></script>      <script>(function(w,d,s,l,i) {w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var  f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true; j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);}) (window,document,'script','dataLayer','GTM-5Q9DBRM');</script>
```

Comment:
Typo ? `<script src="https://help.semmle.com/GAtracking.js"></script>      ` shouldn't still be there!

Reply:
Yep. Not sure what happened here.